README.md

```diff
@@ -1,65 +1,277 @@
-# python-otp4gb
+# OTP4GB-py
 
-This is a port of the [OTP4GB][OTP4GB] tool written by Tom Forth at Open Innovations.
+This tool is a Python wrapper around [Open Trip Planner (OTP) v2](https://docs.opentripplanner.org/en/v2.1.0/)
+which produces cost metrics for public transport routing between given positions in Great Britain.
 
-[OTP4GB]: https://github.com/odileeds/OTP4GB
+This is an updated version of OTP4GB-py produced by Open Innovations (available at
+[github.com/open-innovations/otp4gb-py](https://github.com/open-innovations/otp4gb-py)),
+which itself is a port port of the [OTP4GB](https://github.com/odileeds/OTP4GB)
```

Comment:
"which itself is a **port port**" --> **port**

Reply:
Fixed in 73014cf3fff0ce7ecfc1b0144123aafe469d019c